### DISCUSSION TITLE
PICO HID

### DISCUSSION THREAD

**Discussion by HauTranCong** [earlephilhower/arduino-pico#270] [Q&A] [answered]
Someone can help me, there is no new window. It works with a single key that I just put "Keyboard.press(ctrlKey);" in loop and then I press 'n' in my keyboard so the new window is open. Thanks in advanced

#include <Keyboard.h>
char ctrlKey = KEY_LEFT_CTRL;

void setup() {   
  Serial.begin(9600);
  Keyboard.begin();
}

void loop() {
  delay(1000);
  // new document:
  Keyboard.press(ctrlKey);
  Keyboard.press('n');
  delay(100);
  Keyboard.releaseAll();
  // wait for new window to open:
  delay(1000);
}

**Comment by Andy2No** [comment, marked as answer]
Maybe insert a short delay before Keyboard.press('n');?

Have you established focus on the window?  I've made a page turning pedal with a Leonardo clone and the Mouse library, which is similar.  I have to click on the window I want it to send events to, before I start using it.

**Reply by HauTranCong** [reply]
Yes, I make sure that I click focus on desktop. I trying to put delay before Keyboard.press('n'); now, so how many delay is enough

**Comment by Andy2No** [comment]
Good :)  You're welcome.
